docs/01-module-block-1/02-learning-goals.adoc

```diff
@@ -2,39 +2,83 @@
 
 // tag::DE[]
 [[LZ-1-1]]
+==== LZ 1-1: Daten
 
-- Was sollen die Teilnehmer können?
-   * Informationen als unveränderliche Werte repräsentieren
-   * zentrale Funktionalität als Funktionen formulieren
-   * für diese Funktionen Typen konstruieren
+* die Rolle von systematischer Datenanalyse in der Domänenmodellierung
+  verstehen
+* Informationen systematisch als Daten repräsentieren können
+* den Unterschied zwischen zusammengesetzten und gemischten Daten
+  verstehen
+* die Rolle von Selbstbezügen in der Datenrepräsentation verstehen
+* bei der Datenanalyse Selbstbezüge finden und nutzen können
+* den Unterschied zwischen Objekten und Werten verstehen
+* den Unterschied zwischen funktionalen und objektorientierten
+  Datenstrukturen verstehen
 
-- Was sollen die Teilnehmer verstehen?
-  * den Unterschied zwischen dem objektorientierten und dem funktionalen
-Software-Modell
-  * die Rolle von Typen beim funktionalen Software-Design
-  * Module als Alternative zu Klassen
+[[LZ-1-2]]
+==== LZ 1-2: Funktionen
 
-- Was sollen die Teilnehmer kennen?
-  * Beispiele für funktionale System-Architekturen
+* Funktionen systematisch anhand von Anforderungen konstruieren können
+* den Unterschied zwischen reinen Funktionen und zustandsverändernden
+  Prozeduren beziehungsweise objektorientierten Methoden verstehen
+* systematische Abstraktion anwenden können
+* Funktionen durch Abstraktion komponieren können
 
+[[LZ-1-3]]
+==== LZ 1-3: Typen
+
+* die Rolle von Typen bei der Datenanalyse verstehen
+* Typen benutzen können, um Funktionen systematisch zu konstruieren
+* Datenrepräsentationen in sowohl getypten als auch ungetypten
```

Comment:
Inkonsistente Terminologie: oben steht statisch/dynamisch, hier steht getypt/ungetypt (ich bevorzuge letzteres)

Reply:
@larsh Das ewige Thema :-) Getypt/ungetypt ist technisch richtig, aber was die Softwarearchitektur betrifft verwirrend - es ist ja nicht so, daß beim Architekturdesign "Typen" keine Rolle spielen. Kann ich Dich auf statisch/dynamisch (oder statisch/latent) runterhandeln?

Reply:
Statisch/dynamisch finde ich zwar nicht super, aber akzeptabel, solange es konsistent ist.

Reply:
@larsrh Done in 4bee2b27646df0db52721b4b78a27344fccef833